tools/ui-components/src/index.js

```diff
@@ -1 +1,3 @@
 // Use this file as the entry point for component export
+export * from './button';
+export * from './color-system';
```

Comment:
This color-system thing is just an internal component I created when I needed to display all the colors we have on the screen and compare them. It's not intended to be consumed as a component 😄 

```suggestion
```

Reply:
Oops. I think I just gave away how little I know about storybook 😰

Reply:
It's my bad. I should have marked it internal or something. 

It was that I needed to give our color variables a name, like `red10`, `red20`, etc. However, I couldn't tell which would be which from the hex code, unless I had all of them sitting next to each other on the screen. I then left them there as I thought it would perhaps be useful in the future 😛